CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v1.3.0
+
+This release includes a number of new features and bugfixes like the long-awaited propagation of apps' exit status.
+
+#### New features and UX changes
+
+- Propagate exit status from apps inside the pod to rkt ([#2308](https://github.com/coreos/rkt/pull/2308)). Previously, if an app exited with a non-zero exit status, rkt's exit status would still be 0. Now, if an app fails, its exit status will be propagated to the outside.
```

Comment:
Since it was already mentioned in v1.1.0's changelog, you could add something like:

> This was partially implemented in some stage1 flavors since rkt v1.1.0. This now works in the default coreos flavor.